src/redux/preview.js

```diff
@@ -38,7 +38,8 @@ module.exports.getInitialState = () => ({
     projectStudios: [],
     curatedStudios: [],
     currentStudioIds: [],
-    moreCommentsToLoad: false
+    moreCommentsToLoad: false,
+    projectNotAvailable: false
```

Comment:
FWIW, I prefer using positive names rather than negative names where possible -- e.g., `projectAvailable` here. They make me less confused when mentally parsing lines like `projectNotAvailable: !!action.info` below.

("FWIW" == "I want to say that but I don't expect you to change it")

Reply:
Usually I would agree. The reason I used the negative here is that it feels like it more accurately captures the state. I wanted to avoid "404" or technical language like that which would imply something more specific than what is actually happening. Does that make sense?

Reply:
Sure, I can see that perspective.